lib/server-impl.js

```diff
@@ -21,6 +21,8 @@ function createApp (options) {
         enableRequestLogger: options.enableRequestLogger,
         port: options.port,
         publicFolder: options.publicFolder,
+        preRouterHook: options.preRouterHook,
```

Comment:
Cannot se a reason for why not - @ivarconr ? Changed to use object.assign in another commit.